SparkPost/2web2ui
# TR-1203: Template List Page UX Fix of Whitespace

Refer to [TR-1203](https://jira.int.messagesystems.com/browse/TR-1203) for more information.

### What Changed

- Template list page is initially sorted by last updated column descending
- Slightly modify published tag style — moved bullet to lead and italicized

<img width="604" alt="Screen Shot 2019-04-19 at 3 36 00 PM" src="https://user-images.githubusercontent.com/1335605/56441851-62a70f00-62bc-11e9-8a29-b2e78e8b0ea8.png">

- Removed static widths on metadata field columns to allow name column to grow

<img width="1387" alt="Screen Shot 2019-04-19 at 5 03 56 PM" src="https://user-images.githubusercontent.com/1335605/56444282-0fd25500-62c6-11e9-81a8-cb34da941fb4.png">
<img width="1389" alt="Screen Shot 2019-04-19 at 5 04 14 PM" src="https://user-images.githubusercontent.com/1335605/56444283-106aeb80-62c6-11e9-96a0-4d8557167029.png">

- Bonus: fixed both template and snippet list pages to show error banner if load error occurs


### How To Test
- Navigate to templates list page and observe list is sorted by last updated column and the metadata columns are fixed widths to allow room for long template names
- Find a appteam template that is in published, but has draft changes
- Block /templates request in devtools network tab, then reload the page, the error banner should be displayed
